src/main/java/org/schemaspy/Config.java

```diff
@@ -1220,57 +1221,6 @@ public boolean isLowQuality() {
         return lowQuality;
     }
 
-    /**
-     * Set the level of logging to perform.<p/>
-     * The levels in descending order are:
-     * <ul>
-     * <li><code>severe</code> (highest - least detail)
-     * <li><code>warning</code> (default)
-     * <li><code>info</code>
-     * <li><code>config</code>
-     * <li><code>fine</code>
-     * <li><code>finer</code>
-     * <li><code>finest</code>  (lowest - most detail)
-     * </ul>
-     *
-     * @param logLevel
-     */
-    public void setLogLevel(String logLevel) {
```

Comment:
@danwatt after when we delete setLogLevel parameter is not possible to change application.properties/logging.level.org.schemaspy from command line. 

This was common feature in previous version of SchemaSpy in many cases original error was hidden as long as -loglevel param was not set to higher.

When we will run the SchemaSpy with `java -jar schemaspy.jar --debug` param we will only change `logging.level.root` but not `logging.level.org.schemaspy`

Reply:
I know this is a bit old but...
`--logging.level.org.schemaspy=DEBUG`
works.

Possible solutions:
1, We could use spring placeholder and make it `--loglevel` still the double dash.
2, We could add -loglevel to jcommander and have it injected in to the spring environment.

Reply:
Could just do a `-debug` i Jcommander.